### PR TITLE
Use Travis CI to run tests on all code changes

### PR DESCRIPTION
**Pull request by cclauss**
Every time someone submits a pull request, Travis Continuous Integration service can run the tests on it before it is reviewed.  Travis is always free to opensource projects like this one but the owner of the this repo would need to go to https://travis-ci.org/profile and flip the repository switch on to enable testing.

### CODE REVIEW

**Comment by lorenzb** [comment]
Thanks for your PR @cclauss!

Travis jobs are [automatically killed after 50 minutes](https://docs.travis-ci.com/user/customizing-the-build#Build-Timeouts) and our full test suite typically takes longer than that.

A potential solution might be to split the travis build into a few different jobs.

**Comment by lorenzb** [comment]
Travis is enabled

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+language: python
+cache: pip
+python:
+    - 2.7
```
Comment (lorenzb) [inline .travis.yml]: We only use Python >=3.6

**Comment by cclauss** [comment]
I will need your guidance on why the __tests are not running__:
* https://travis-ci.org/projectchicago/gastoken/builds/351671585#L576-L728

**Comment by cclauss** [comment]
This one sets a new record!  https://travis-ci.org/projectchicago/gastoken/builds/351671585#L499

;-)

**Comment by lorenzb** [comment]
Tests are failing because the solidity compiler `solc` isn't available in the travis environment.
Installation instructions are [here](https://solidity.readthedocs.io/en/v0.4.20/installing-solidity.html).

**Comment by cclauss** [comment]
['elopio' installed](https://travis-ci.org/projectchicago/gastoken/builds/351692281#L726) but it does not seem to help.

**Comment by cclauss** [comment]
@lorenzb When you get a minute, can you take a look at https://travis-ci.org/projectchicago/gastoken/builds/351692281 and let me know what is missing?

**Pull request closed without merging**